pipelines/datasets/world_wil_wid/constants.py

```
@@ -0,0 +1,112 @@
+
+    # TRAP 1 -- the archive carries a pair of header-only artifacts dated
+    # 2024-02-14, `WID_data_Al.csv` (47 bytes) and `WID_metadata_Al.csv`
+    # (168 bytes), whose names differ from Albania's real `WID_data_AL.csv`
+    # (19.7 MB) and `WID_metadata_AL.csv` (2.1 MB) only by case. On a
+    # case-insensitive filesystem -- macOS/APFS, the default -- extracting the
+    # archive overwrites BOTH of Albania's files with the stubs, and the
+    # country vanishes with no error. Never extract these two members.
+    JUNK_MEMBERS = frozenset({"WID_data_Al.csv", "WID_metadata_Al.csv"})
+
+    # Table slugs, in publication order.
+    TABLES = ["indicator", "series", "country", "dicionario"]
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)